test/axis-interval.html

```diff
@@ -534,8 +540,68 @@ <h1>Test xAxis.axisTick.interval</h1>
 
 
 
+    <script>
+
+        require([
```

Comment:
Please add more test cases to test boundary situations. You may start with the cases in my above questions.